### DISCUSSION TITLE
instagram not blocking

### DISCUSSION THREAD

**Discussion by arslan823** [AdguardTeam/AdGuardHome#2864] [Q&A] [answered]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * 
* **How did you install AdGuard Home:**
  * 
* **How did you setup DNS configuration:**
  * 
* **If it's a router or IoT, please write device model:**
  * 
* **CPU architecture:**
  * 
* **Operating system and version:**
  * 

### Screenshots

<details><summary>Screenshot:</summary>

</details>

**Comment by ameshkov** [comment, marked as answer]
DNS level blocking is not enough to block ads in Instagram as their ads are embedded into the feed.

**Reply by arslan823** [reply]
blocking the Instagram service does not work. I didn't mean advertising. sorry for my English

**Reply by ameshkov** [reply]
Could it be that Instagram cached the DNS response and uses previously received IP addresses? Try clearing the app cache.

**Reply by arslan823** [reply]
the cache was cleared by rebooting the device. the service is not blocked specifically on the mobile app Instagram

**Reply by ameshkov** [reply]
I am not sure that rebooting device is enough. What if you try reinstalling Instagram? This way it will be cleared for sure.

**Reply by quaintdev** [reply]
Or they might have embedded IP addresses directly into the app
